Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+pipeline {
+
+    options {
+        timestamps()
+        timeout(time: 45, unit: 'MINUTES', activity: true)
+    }
+
+    agent {
+        node {
+            label "cloud-ccp-ci"
+        }
+    }
+    environment {
+        SOCOK8S_ENVNAME = "cloud-socok8s-${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
+        OS_CLOUD = "engcloud-cloud-ci"
+        KEYNAME = "engcloud-cloud-ci"
+        DELETE_ANYWAY = "YES"
+    }
+
+    stages {
+        stage('Show environment information') {
+            steps {
+                sh 'printenv'
+            }
+        }
+
+        stage('Deploy everything') {
+            steps {
+                sh "./run.sh"
+            }
+        }
+    }
+
+    post {
+        failure {
+            script {
+                echo "It failed. We want to keep the env here?!"
```

Comment:
I would change script by this script:
```
      script {
        if (env.hold_instance_for_debug == 'true') {
          echo "You can reach this node by connecting to its floating IP as root user, with the default password of your image."
          timeout(time: 3, unit: 'HOURS') {
               input(message: "Waiting for input before deleting the ccp env.")
          }
        }
```

Advantage: When ppl want to hold an instance for debugging purposes, they can just set a variable in environment just a little above, in the same PR. Super easy, and everything done in advance for the dev.

Reply:
Done